client/my-sites/invites/invite-accept/index.jsx

```diff
@@ -98,15 +109,20 @@ let InviteAccept = React.createClass( {
 		page( '/' );
 	},
 
+	signInLink() {
+		return config( 'login_url' ) + '?redirect_to=' + encodeURIComponent( window.location.href );
+	},
+
 	renderForm() {
-		if ( ! this.state.invite ) {
+		const { invite, user, matchEmailError } = this.state;
+		if ( ! invite ) {
 			debug( 'Not rendering form - Invite not set' );
 			return null;
 		}
 		debug( 'Rendering invite' );
-		return this.state.user
-			? <LoggedIn invite={ this.state.invite } redirectTo={ getRedirectAfterAccept( this.state.invite ) } decline={ this.decline } user={ this.state.user } />
-			: <LoggedOut { ...this.state.invite } redirectTo={ getRedirectAfterAccept( this.state.invite ) } decline={ this.decline } />;
+		return user
+			? <LoggedIn invite={ invite } redirectTo={ getRedirectAfterAccept( invite ) } decline={ this.decline } user={ user } signInLink={ this.signInLink() } matchEmailError={ matchEmailError } />
```

Comment:
This line is starting to get pretty lengthy. It might make sense to break it up like this:

```
<LoggedIn 
  invite={ this.state.invite } 
  redirectTo={ getRedirectAfterAccept( this.state.invite ) }
...
...
/>
```